tests/integration/interface_common_test.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import libnmstate
+
+from .testlib.statelib import INTERFACES
+
+
+def test_set_empty_iface_description(eth1_up):
+    desired_state = {INTERFACES: [{'name': 'eth1', 'description': ''}]}
+    libnmstate.apply(desired_state)
+
```

Comment:
You need to check something so it will be clear what is the scenario expectation.
How about reporting back the current state and asserting the expected missing key?